.gitmodules

```diff
@@ -25,3 +22,9 @@
 [submodule "src/extensions/default/JSLint/thirdparty/jslint"]
 	path = src/extensions/default/JSLint/thirdparty/jslint
 	url = https://github.com/peterflynn/JSLint.git
+[submodule "src/extensions/default/JavaScriptCodeHints/node/node_modules/acorn"]
+	path = src/extensions/default/JavaScriptCodeHints/node/node_modules/acorn
+	url = https://github.com/marijnh/acorn.git
+[submodule "src/extensions/default/JavaScriptCodeHints/node/node_modules/tern"]
+	path = src/extensions/default/JavaScriptCodeHints/node/node_modules/tern
+	url = https://github.com/marijnh/acorn.git
```

Comment:
FYI `.gitmodules` now conflict with #11994 (and the `url` is wrong, should be `https://github.com/ternjs/tern.git`, but that will be fixed by fixing the conflict too)